scripts/build.js

```diff
@@ -20,7 +20,9 @@ if (FLAGS.indexOf('--rinkeby') !== -1) {
 } else if (FLAGS.indexOf('--aura') !== -1) {
   network = 'aura'
 }
-process.env.ETHEREUM_NETWORK =  network
+
+// environment variable trumps passed in flag
+if (network && !process.env.ETHEREUM_NETWORK) { process.env.ETHEREUM_NETWORK = network }
```

Comment:
It’s looking like there is a logical issue here in that if the network is set by environment variable then the console log below will report incorrectly.

Reply:
makes sense to me, flag trumps environment variables. code updated.